README.md

```diff
@@ -4,7 +4,12 @@ This is a seed project to get us started with prototyping.
 
 ## Installation
 
-First install the toolchain with [NPM](https://www.npmjs.org/):
+First install the command line tools globally with [NPM](https://www.npmjs.org/):
+```shell
+npm install --global gulp uglify-js2
```

Comment:
Yes, you don't need to ask users install global dependency. I [wrote](http://azimi.me/2014/12/18/npm-start.html) about it in my blog. :)